tests/test_person_relations_phase1.py

```
@@ -0,0 +1,277 @@
+        }
+
+        # Check 25 builtin relation types are seeded
+        cursor = conn.execute(
+            "SELECT relation_type_id, slug, forward_label, reverse_label, directionality, is_builtin, is_active FROM person_relation_types WHERE is_builtin = 1;"
+        )
+        builtin_rows = cursor.fetchall()
+        assert len(builtin_rows) == 25
+        assert len(builtin_rows) == len(BUILTIN_RELATION_TYPES)
+
+        for row in builtin_rows:
+            rt_id, slug, f_label, r_label, directionality, is_builtin, is_active = row
```

Ruff (0.16.3)

[warning] Unpacked variable `f_label` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)